### PR TITLE
Update example repo after renaming dataplane -> forwarder

### PR DESCRIPTION
**Pull request by denis-tingaikin**
Signed-off-by: denis-tingajkin <denis.tingajkin@xored.com>``

### CODE REVIEW

**Review by nickolaev — COMMENTED** [review]
It looks like the strings in the go code are jist wrong. We should just replace them with relevant ones.

**Comment by denis-tingaikin** [comment]
@nickolaev can you provide an example of wrong strings in the go code?

File: examples/proxy/proxy-nsc/cmd/nsc.go [hunk 1]
```diff
@@ -105,11 +105,11 @@ func proxyHost() string {
 
 func main() {
 	// Init the tracer
-	closer := jaeger.InitJaeger("vppagent-dataplane")
+	closer := jaeger.InitJaeger("vppagent-forwarder")
```
Comment (nickolaev) [inline examples/proxy/proxy-nsc/cmd/nsc.go]: "vppagent-forwarder" --> "proxy-nsc"

File: examples/proxy/proxy-nsc/cmd/nsc.go [hunk 2]
```diff
@@ -105,11 +105,11 @@ func proxyHost() string {
...
 
 	defer func() { _ = closer.Close() }()
 
-	span := spanhelper.FromContext(context.Background(), "Start.VPPAgent.Dataplane")
+	span := spanhelper.FromContext(context.Background(), "Start.VPPAgent.Forwarder")
```
Comment (nickolaev) [inline examples/proxy/proxy-nsc/cmd/nsc.go]: "Start.VPPAgent.Forwarder" --> "Start.Proxy.NSC"

File: examples/vpp-icmp/vppagent-client/cmd/client.go [hunk 3]
```diff
@@ -64,11 +64,11 @@ func main() {
 	c := tools.NewOSSignalChannel()
 
 	// Init the tracer
-	closer := jaeger.InitJaeger("vppagent-dataplane")
+	closer := jaeger.InitJaeger("vppagent-forwarder")
```
Comment (nickolaev) [inline examples/vpp-icmp/vppagent-client/cmd/client.go]: "vppagent-forwarder" --> "vppagent-client"

File: examples/vpp-icmp/vppagent-client/cmd/client.go [hunk 4]
```diff
@@ -64,11 +64,11 @@ func main() {
...
 
 	defer func() { _ = closer.Close() }()
 
-	span := spanhelper.FromContext(context.Background(), "Start.VPPAgent.Dataplane")
+	span := spanhelper.FromContext(context.Background(), "Start.VPPAgent.Forwarder")
```
Comment (nickolaev) [inline examples/vpp-icmp/vppagent-client/cmd/client.go]: "Start.VPPAgent.Forwarder" --> "Start.VPPAgent.Client"

**Review by nickolaev — CHANGES_REQUESTED** [review]
Added suggested string substitutions.

**Comment by denis-tingaikin** [comment]
@nickolaev got it. Thanks, fixed

**Review by nickolaev — APPROVED** [review]

**Pull request merged**